packages/compass-aggregations/src/components/atlas-no-results.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
NIT: is this AtlasSearchNoResults, missingAtlasSearchIndex ...?

Reply:
Any atlas-only stage. I didn't fix that bug. I'm not even sure if it is a bug since more stages than just $search relate to atlas search.

Reply:
isAtlasOnly() seems to map to $search, $searchMeta, $vectorSearch now. Which actually sounds about right.

Reply:
then this was probably messed up before, i think we should change it. 

This behaviour is not intended for any atlas stage. I can totally see us adding a new atlas only stage inside devtools-shared that is not a search feature, and get this weird error which is only meant for search.

Reply:
Updated now. Let me know what you think.